src/sourmash/cli/tax/genome.py

```diff
@@ -7,21 +7,24 @@
 The 'tax genome' command reads in genome gather result CSVs and reports likely
 classification for each query genome.
 
-By default, classification uses a containment threshold of 0.1, meaning at least
-10 percent of the query was covered by matches with the reported taxonomic rank and lineage.
-You can specify an alternate classification threshold or force classification by
-taxonomic rank instead, e.g. at species or genus-level.
+By default, classification uses a containment threshold of 0.1,
+meaning at least 10 percent of the query was covered by matches with
+the reported taxonomic rank and lineage.  You can specify an alternate
+classification threshold or force classification by taxonomic rank
+instead, e.g. at species or genus-level.
 
 The default output format consists of five columns,
- `query_name,status,rank,fraction,lineage`, where `fraction` is the fraction
- of the query matched to the reported rank and lineage. The `status` column
+ 'query_name,status,rank,fraction,lineage', where 'fraction' is the fraction
+ of the query matched to the reported rank and lineage. The 'status' column
  provides additional information on the classification, and can be:
-  - `match` - this query was classified
-  - `nomatch`- this query could not be classified
-  - `below_threshold` - this query was classified at the specified rank,
+  - 'match' - this query was classified
+  - 'nomatch'- this query could not be classified
+  - 'below_threshold' - this query was classified at the specified rank,
```

Comment:
```suggestion
  - 'below_threshold' - this query was classified at the desired rank,
```

(I think this only happens when forcing classification at a specific rank with `--rank`)

Reply:
actually, idk which of this is actually better